packages/vite/src/node/utils.ts

```diff
@@ -118,7 +118,7 @@ export const isJSRequest = (url: string): boolean => {
   return false
 }
 
-const importQueryRE = /(\?|&)import(?:&|$)/
+const importQueryRE = /(\?|&)import=(?:&|$)/
```

Comment:
Sorry for the miscommunication. We shouldn't change from `import` to `import=`, there are projects like SvelteKit that are generating the import without the `=`. I think that this PR should only update the regexs to accept both with and without the `=`. This should resolve the issue with Vercel (and possibly other platforms) without introducing issues in the ecosystem.

Reply:
Ah - didn't realize there were other projects that were dependent on this. Will update now :)